commerce_coordinator/apps/ecommerce/views.py

```diff
@@ -95,6 +95,8 @@ def get(self, request):  # pylint: disable=inconsistent-return-statements
             'edx_lms_user_id': request.query_params.get('edx_lms_user_id'),
             # TODO: email should be taken from request.user.email once we go live.
             'email': request.query_params.get('email'),
+            'first_name': request.user.first_name,
+            'last_name': request.user.last_name,
```

Comment:
`request.user` will be a worker user, Not an actual user. It will point to the actual user in the future when these calls will be made from the front end. For Now, worker first_name and last_name should be fine

Reply:
passing dummy values.